packages/editor/src/components/post-publish-button/index.js

```diff
@@ -49,6 +48,10 @@ export class PostPublishButton extends Component {
 				this.setState( {
 					entitiesSavedStatesCallback: () => callback( ...args ),
 				} );
+				this.props.setCloseEntitiesCallback(
+					() => this.closeEntitiesSavedStates
```

Comment:
```suggestion
					this.closeEntitiesSavedStates
```

It's already bound.

Reply:
Similar to the above, when setting a function in `useState`, if we were to only pass the function it would try to set that functions return as the value on state, erroring with `closeEntitiesCallback` is not a function.

Reply:
Right, because this is the actual setter, not a proxy. A comment to clarify would be useful.